verilog/analysis/checkers/signal_name_style_rule.h

```diff
@@ -56,7 +56,9 @@ class SignalNameStyleRule : public verible::SyntaxTreeLintRule {
 
   using Matcher = verible::matcher::Matcher;
 
-  Matcher matcher_ = NodekPortDeclaration();
+  Matcher matcher_port_ = NodekPortDeclaration();
+  Matcher matcher_net_ = NodekNetDeclaration();
+  Matcher matcher_data_ = NodekDataDeclaration();
```

Comment:
Is that acceptable to use multiple matchers in one lint rule? Or maybe there is a way to combine them into one matcher? I am not sure if the latter would even make sense as the handling would be somewhat different anyway

Reply:
There no rule limiting the use of matchers, they are implementation details.
There are some matcher operators like Or and Unless (Not), but you don't have to use them if you feel it is simpler to combine/refine matching differently.